tests/init.go

```diff
@@ -0,0 +1,60 @@
+package tests
+
+import (
+	"flag"
+	"regexp"
+	"strings"
+
+	"github.com/ipfs/gateway-conformance/tooling/specs"
+)
+
+type specsFlag string
+
+func (s *specsFlag) String() string {
+	return string(*s)
+}
+
+func (s *specsFlag) Set(value string) error {
+	names := strings.Split(value, ",")
+	var only, enable, disable = []specs.Spec{}, []specs.Spec{}, []specs.Spec{}
+	for _, name := range names {
+		spec, err := specs.FromString(regexp.MustCompile(`^[-+]`).ReplaceAllString(name, ""))
+		if err != nil {
+			return err
+		}
+		if strings.HasPrefix(name, "+") {
+			enable = append(enable, spec)
+		} else if strings.HasPrefix(name, "-") {
```

Comment:
If a spec is prefixed with `-`, it should be disabled. This can be used to disable tests for specs that are enabled by default (those mature enough).

For example, we can disable `subdomain-gateway` tests while still testing all the other specs that are enabled by default by providing `-specs=-subdomain-gateway` flag.